captum/optim/_models/inception_v1.py

```diff
@@ -50,7 +59,15 @@ def googlenet(
         else:
             state_dict = torch.load(model_path, map_location="cpu")
         model.load_state_dict(state_dict)
-        model_utils.replace_layers(model)
+
+        if (
+            replace_relus_with_redirectedrelu
+            and not replace_nonlinears_with_linear_equivalents
+        ):
+            model_utils.replace_layers(model)
+        elif replace_nonlinears_with_linear_equivalents:
+            model_utils.max2avg_pool2d(model)
+            model_utils.ignore_layer(model, model_utils.ReluLayer)
```

Comment:
Thank you for making the changes, @ProGamerGov! I was thinking that instead of making the replacements we can create the instance of `InceptionV1` with an argument e.g. `use_linear_modules_only`. In this case we can avoid doing replacements and calling `ignore_layer` function. Current implementation is not very different from the original implementation. The replacement piece looks like a model patch and I was thinking to construct the model with linear modules if there is such a kwargs specified. Basically, skipping relus and using AveragePool instead of maxpool when constructing the model.
Could we do that ? `https://github.com/pytorch/captum/blob/441b751150153fc423260e4419f8d7ac90626e6c/captum/optim/_models/inception_v1.py#L53`

Reply:
@NarineK I've made the changes!